src/main/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountResponseModel.java

```diff
@@ -10,8 +10,8 @@
 @AllArgsConstructor(access = AccessLevel.PUBLIC)
 public class AccountResponseModel {
     private String accountId;
+    private String userId;
     private String username;
-    private String password;
     private String email;
     private String city;
 }
```

Comment:
What is userId used for?

Reply:
Good question, there will be 2 different ids that will be supported inside the account entity, one is for the accountId, which is the random UUID that will be used for inner database calls, and the userId which is the authentification ID you get when creating an account.

Reply:
Okay thanks for the clarification.